README.md

```diff
@@ -57,10 +57,10 @@ of the ForMath EU FP7 project (2009-2013). It has two parts:
   - [Refinements for free!](https://hal.inria.fr/hal-01113453/document) doi:[10.1007/978-3-319-03545-1_10](https://doi.org/10.1007/978-3-319-03545-1_10)
   - [A Coq Formalization of Finitely Presented Modules](https://hal.inria.fr/hal-01378905/document) doi:[10.1007/978-3-319-08970-6_13](https://doi.org/10.1007/978-3-319-08970-6_13)
   - [Formalized Linear Algebra over Elementary Divisor Rings in Coq](https://hal.inria.fr/hal-01081908/document) doi:[10.2168/LMCS-12(2:7)2016](https://doi.org/10.2168/LMCS-12(2:7)2016)
-  - [A refinement-based approach to large scale reflection for algebra](https://hal.inria.fr/hal-01414881/document) 
```

Comment:
@clayrat the spaces here are a consequence of our (somewhat lacking) toolchain that generates boilerplate from the `meta.yml` file: https://github.com/coq-community/templates

Hence, there is not really any point in removing these spaces, since they will reappear the next time this file is regenerated from the template. Also, if you want to change the content, you should change `meta.yml` and regenerate rather than changing the file (`README.md`) directly.

Reply:
Oh, the actual change here was adding the commas in "`rank`  `strassen` `toomcook`" and the spaces are just from my autolinter. I should add those commas to the `yml` as well, indeed.